changelog.d/PR342.bugfix

```diff
@@ -0,0 +1 @@
+Handle CORS for GetValidated3pidServlet. Endpoint /3pid/getValidated3pid returns valid CORS headers.
```

Comment:
```suggestion
Handle CORS for `GetValidated3pidServlet`. Endpoint `/3pid/getValidated3pid` returns valid CORS headers.
```

These changelog files end up [in CHANGELOG.md](https://github.com/matrix-org/sydent/blob/main/CHANGELOG.md). So feel free to use markdown!